plutus-metatheory/src/Opts.hs

```diff
@@ -6,27 +8,13 @@ import Data.Text qualified as T
 import Data.Text.IO qualified as T
 import Options.Applicative
 
-data Input = FileInput T.Text | StdInput deriving stock (Show, Read)
-
-
-fileInput :: Parser Input
-fileInput = FileInput <$> strOption
-  (  long "file"
-  <> short 'f'
-  <> metavar "FILENAME"
-  <> help "Read from file" )
+import Common
+import Parsers
 
-stdInput :: Parser Input
-stdInput = flag' StdInput
-  (  long "stdin"
-  <> help "Read from stdin" )
+-- The different evaluation modes of plc-agda
+data EvalMode = U | TL | TCK | TCEK deriving stock (Show, Read)
```

Comment:
`L` and `CK` modes are removed?

Reply:
Those modes are longer supported (there's no evaluator code for them).